### ISSUE TITLE
Image randomly rotated

### ISSUE THREAD

**Issue by fredfp** [sksamuel/scrimage#89]
Using version 2.0.2, loading the attached image (using `val img = Image.fromStream`) and writing it to file using `img.output("chuck-is-sea-sick.jpg")(JpegWriter.Default)` without any other operations sometimes outputs an image rotated 90 degrees clockwise (see attached).

input:
![chuck](https://cloud.githubusercontent.com/assets/3804696/8689415/10c08d70-2a5c-11e5-8ae4-df6839e70b34.jpg)

output I sometimes get:
![chuck-is-sea-sick](https://cloud.githubusercontent.com/assets/3804696/8689439/61c39050-2a5c-11e5-8cc2-2fd95ce1b316.jpg)

**Comment by fredfp** [comment]
I tried about 20 times, but this doesn't happen with version 1.4

**Comment by sksamuel** [comment]
@fredfp do you get this if you just run the same thing in a loop - the non-determinism that is ?

**Comment by fredfp** [comment]
I didn't implement a loop. The image above is the output of a test case that I run multiple times until it generates the rotated image i.e., I run `test` multiple times in sbt.

**Comment by sksamuel** [comment]
What I mean is, if its random, and you run your code in a loop then you
should get random output ?

**Comment by fredfp** [comment]
I understood, but didn't try it... I'm extremely busy these days, I'll try to find a couple minutes to give it a try.

**Comment by azickh** [comment]
I have also seen this happen when the height is greater than the width.

**Comment by azickh** [comment]
Please, fix it,
Thanks!

**Comment by sksamuel** [comment]
I've written a test on master, 
```scala
      for ( k <- 1 to 1000 ) {
        val image = Image.fromResource("/chuck.jpg")
        image.width shouldBe 319
        image.height shouldBe 397
        image.output(s"chuck$k.jpg")(JpegWriter.Default)
      }
```

And all 1000 images outputted correctly. All I can think of is some strangeness with the iphone rotation detection. Can you try the latest beta @fredfp @azickh

**Comment by sksamuel** [comment]
Ok I can reproduce this now. It seems that the test either fails completely on all loads, or runs completely.

**Comment by sksamuel** [comment]
Fixed, will release another 2.1 beta.

**Comment by azickh** [comment]
Ok, thanks Stephan,
It's not reproduced if ignor ImageMetadata, as workaround i use it.
